packages/coil-extension/manifest.json

```diff
@@ -32,5 +32,5 @@
       "run_at": "document_start"
     }
   ],
-  "permissions": [ "webNavigation", "<all_urls>" ]
+  "permissions": [ "webNavigation", "<all_urls>", "tabs" ]
```

Comment:
Could maybe look into build target specific, browser permissions, i.e. only adding the "tabs" permission when Safari is the build target. The "accept-monetization" header implemented via declarativeNetRequest also needs some system for declaring permissions for specific build targets.